### DISCUSSION TITLE
Custom Home Page with Slideshow?

### DISCUSSION THREAD

**Discussion by Phil89a** [nunocoracao/blowfish#1460] [Q&A]
I am trying to create a homepage so that I can fill the content area (below the header and above the footer) with an image that changes after a number of seconds. I have looked at the Carousel shortcode, but I can't see how to get the image to fill the content area (very new to all of this).

Can anyone help me get going?

**Comment by MaxiCabanas** [comment]
Were you able to get the carousel working on the home page? I'm trying to do the same and can't make the images change after x time.

I did achieve what you need by wrapping the images with :

```
<article class="max-w-full prose dark:prose-invert">
    <div class="relative">
        <div class="absolute inset-x-0 bottom-0 h-1/2 bg-gray-100"></div>
        <div class="mx-auto max-w-7xl p-0">
            <div class="relative sm:overflow-hidden">
                 [[CAROUSEL]]
            </div>
        </div>
    </div>
</article>
```
